keymanager/src/client/remote.rs

```diff
@@ -584,4 +596,102 @@ impl KeyManagerClient for RemoteClient {
             .await
             .map_err(|err| KeyManagerError::Other(err.into()))
     }
+
+    async fn state_key(
```

Comment:
Wanted to exclude the prefix so that method names remain the same if we split this client into two parts, one for secrets and one for churp. Will add prefix for now.